packages/text-field/theme/lumo/vaadin-input-field-shared-styles.js

```diff
@@ -172,14 +119,6 @@ registerStyles(
     }
 
     /* Invalid style */
-    :host([invalid]) [part='input-field'] {
-      background-color: var(--lumo-error-color-10pct);
-    }
-
-    :host([invalid]) [part='input-field']::after {
-      background-color: var(--lumo-error-color-50pct);
-    }
```

Comment:
This CSS is moved here:
https://github.com/vaadin/web-components/blob/91bfb8acc231ec7747bbebbfe2add5b83e647e00/packages/input-container/theme/lumo/vaadin-input-container-styles.js#L72-L78